process-engine/process_engine
# fix undefined token payload

## What did you change?

During review of https://github.com/process-engine/bpmn-studio/pull/573 we found errors when a process was started with an undefined token payload.

## How can others test the changes?

* run the tests mentioned in https://github.com/process-engine/bpmn-studio/pull/573
* if you start the `Prozess erstellen` process, you can inspect the `FlowNodeInstance` table in the database
* it should have entries

## PR-Checklist

Please check the boxes in this list after submitting your PR:
